vizro-core/docs/pages/tutorials/explore-components.md

```diff
@@ -1,41 +1,29 @@
 
-By the end of this tutorial, you will have gained a deep understanding of Vizro dashboards: their structure, adding
-functionality, and configuring layouts to suit your specific needs.
+If you haven't yet done so, you may want to review the [first dashboard tutorial](../tutorials/first-dashboard.md) before starting on this one.
 
-This step-by-step guide is designed to equip you
-with the knowledge and skills required to create your own dashboards using Vizro.
+## 1. Install Vizro and get ready to run your code
 
-In case you are completely new, you may want to review our [Get started](../tutorials/first-dashboard.md) tutorial first.
+To get started with this tutorial, make sure you have [installed Vizro](../user_guides/install.md), and can run the dashboard code [within a Jupyter Notebook cell](../first_dashboard/#2-open-a-jupyter-notebook), or from a Python script.
 
-## Let's get started!
+## 2. Create a first dashboard page
 
-### 1. Install Vizro and get ready to run your code
+In this section we create a new [`Page`][vizro.models.Page] called `first_page`.
 
-In order to get started with this tutorial, you should have Vizro installed, and be ready to evaluate your code, either by
-pasting it into a Jupyter notebook cell, or by running it from a python script.
+The foundation of every Vizro dashboard is a [`Page`][vizro.models.Page] object. A page uses a set of component types to display the content of the page. These components can be objects such as [`Graph`][vizro.models.Graph], [`Table`][vizro.models.Table], [`Card`][vizro.models.Card], [`Button`][vizro.models.Button], [`Container`][vizro.models.Container], or [`Tabs`][vizro.models.Tabs].
 
+### 2.1. Add the first figure
 
-### 2. Create first dashboard page
+Vizro uses [`Graph`][vizro.models.Graph] objects and [Plotly Express functions](https://plotly.com/python-api-reference/plotly.express.html) to build different types of [figures](https://plotly.com/python-api-reference/generated/plotly.graph_objects.Figure.html).
 
-At the core of every dashboard is a [`Page`][vizro.models.Page] object,
-which serves as its foundation. Think of it as a blank canvas where you can arrange and customize elements like charts and text.
+The code below shows the steps necessary to add a box plot to the page:
 
-In this first section, you will explore how to create a page with different elements and delve into the layout
-configuration.
+1. Add a Vizro [`Graph`](vizro.models.Graph) to the `components` list.
+2. Assign it a unique id (`box_cont`).
+3. Add a [`plotly.express.box`](https://plotly.com/python-api-reference/generated/plotly.express.box.html#plotly.express.box) figure to the list of components.
```

Comment:
I don't know why we have this step in here. But generally we do not need to provide an ID unless we need to target that chart in some kind of interactions. I would therefore remove the ID from all examples where this is not the case.
```suggestion
2. Add a [`plotly.express.box`](https://plotly.com/python-api-reference/generated/plotly.express.box.html#plotly.express.box) figure to the list of components.
```

Reply:
I can't comment on that here but i would also remove it from the code in the examples here